NextCenturyCorporation/itm-evaluation-server
# ITM-535 scenario runner for dry run

**SRY FORGOT TO MENTION.  Make a new config.ini from config.ini.template**

**LMK if you don't have the updated scenarios, I'll send it in Teams**

1. Updates SoarTech runs to only run the correct alignment target for the corresponding SoarTech scenario.  i.e QOL alignment targets are only for SoarTech QOL scenarios, and same with VOL. This should also half the number of runs from 240 to 120 for SoarTech.
2. Adds evalName/evalNumber to history json
3. Updated filename convention:
  1.  evaluation_type - scenario_id - scene_type - alignment_type-adm-name-timestamp
  2. timestamp was updated from "Aug19" to "20240819"

To Test:
python itm_minimal_runner.py --name foobar --session soartech  

To make sure I didn't break anything run:
python itm_minimal_runner.py --name foobar --session eval  
python itm_minimal_runner.py --name foobar --session adept 

I'll continue testing.
